### PR TITLE
Added Visual Studio Code

### PR DESCRIPTION
**Pull request by virgil**
Title says it all.

### CODE REVIEW

**Comment by jjmstark** [comment]
@maurelian I think we've been doing "first in first out" for ordering of tools on this page - should it be at the top, or should it be below EthFiddle? The fact that it's a notable professional tool might suggest it should be higher on the list.

**Comment by virgil** [comment]
I personally suggest putting it at the top.  I think Some level of expert judgement in the ordering is fine.  And bluntly, visual studio is already the most powerful IDE, and Microsoft has agreed to continue developing it.

**Comment by maurelian** [comment]
At the top seems reasonable to me. "Add to the bottom" is just a decent tie-breaker in the absence of other strong ordering indicators.

File: docs/build/index.md [hunk 1]
```diff
@@ -73,6 +73,12 @@ Ethereum has a large and growing number of tools to help developers build, test,
 
 ## Integrated Development Environments (IDEs)
 
+
+### Visual Studio Code *Professional cross-platform IDE with official Ethereum support.*
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Azure Blockchain Workbench plugin](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft-azure-blockchain.azure-blockchain-workbench?tab=Overview)
+- [Code samples](https://github.com/Azure-Samples/blockchain/blob/master/blockchain-workbench/application-and-smart-contract-samples/readme.md)
```
Comment (maurelian) [inline docs/build/index.md]: This is a powerful third party solidity extension (by a colleague of mine): https://github.com/tintinweb/vscode-solidity-auditor

@virgil, what's your take on adding it as a bullet point here, or would it be best as a stand alone item?

**Comment by virgil** [comment]
Sure add the bullet point.

File: docs/build/index.md (hunk 1 above)
Comment (virgil) [reply docs/build/index.md]: Stand alone.  Visual Studio is going to be big.  And it'll make Microsoft happy that they are getting noticed.

**Comment by virgil** [comment]
What's needed to get this accepted?  I want to see a happy Microsoft so they'll keep making baller dev tools for us.

**Comment by jjmstark** [comment]
@maurelian anything additional needed to merge this?

**Comment by maurelian** [comment]
No objection to merging. 

However there's a merge conflict in a build file, which I had to fix locally, so we should merge this instead: https://github.com/ethereum/ethereum-org-website/pull/96

Also, are we merging to staging or master now?

**Comment by maurelian** [comment]
Replaced by #96.

**Pull request closed without merging**